plugins/gopass.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gopass
+spec:
+  version: "v0.0.3"
+  platforms:
+    - selector:
+        matchExpressions:
+          - {key: os, operator: In, values: [darwin, linux]}
```

Comment:
The script is in pure bash. I think it would also make sense to allow installation on Windows and mention bash as a pre-requisite (which you already do).

Reply:
Actually, windows has bash support in newest windows 10 build. I've enabled it. Not sure about the ending though. I will probably revisit later on as gopass support for windows itself is an ongoing topic.

Reply:
I'm a bit puzzled now. The manifest has a windows section which also make the CI test fail. Was that intentional?

Reply:
No it shouldn't fail. I made another attempt to fix the build.